pika/heartbeat.py

```diff
@@ -9,37 +9,32 @@
 
 class HeartbeatChecker(object):
     """Checks to make sure that our heartbeat is received at the expected
```

Comment:
==> "Send heartbeats to the broker and checks to make sure that broker's heartbeat is received before the expected timeout expires." ?